packages/orbit-design-tokens/src/convertRgbaToHex.ts

```diff
@@ -0,0 +1,12 @@
+export type RgbaToHex = (red: number, green: number, blue: number, alpha: number) => string;
+
+const convertRgbaToHex: RgbaToHex = (r, g, b, a) => {
```

Comment:
I would rewrite that function to accept only one parameter (rgba). 

and do something like that: 
```suggestion
const convertRgbaToHex = (rgba: string) => {
  const [r, g, b, a] = rgba
    .replace("rgba(", "")
    .replace(")", "")
    .split(",")
    .map((v) => parseInt(v, 10));

  return `#${((1 << 24) + (r << 16) + (g << 8) + b).toString(16).slice(1)}`;
};
```

Reply:
I was considering something like that at first, but I think the readability of the code is worse. I personally prefer to have lines of code that I can read and understand, if possible. Unless there is a major gain in performance or effectiveness, I don't see a big advantage in changing the code… WDYT?

Reply:
It's a bit of all. Also, it's not bulletproof, as it is not considering when it is just `rgb` (and not `rgba`). And it is missing the alpha conversion for the hex. It can be added, but the function won't be this cleaner, and readability will be (even more) worse, imo

Reply:
It's a bit of all. Also, it's not bulletproof, as it is not considering when it is just `rgb` (and not `rgba`). And it is missing the alpha conversion for the hex. It can be added, but the function won't be this cleaner, and readability will be (even more) worse, imo

Reply:
anyway, my point is, that the function can look whatever you would like it to look, but should accept the string instead of several parameters. So it can be reused next time somewhere without implementing additional logic :)

Reply:
Ah, got it, that can be done, for sure 😄

Reply:
Eh… I tried to, but the `value` in DesignTokenList, which will be passed to this function, is of type `TokenValueType`, which is string or number. So I either do some verification beforehand, or I convert to string, but in both scenarios I thinks that is overcomplicating.
Also, if I call this function `convertRgbaToHex` and I pass a random string, I am not returning an Hex (should I return the same string or throw some error?), so the name is misleading, for it to be in such abstracted level. I think it should be the caller's responsibility to pass the right parameters, that is why I deconstructed it into multiple arguments at first. WDYT?